rfcs/accessibility/accessibility.md

```diff
@@ -0,0 +1,74 @@
+# RFC: Accessibility
+
+---
+
+Contributors:
+@jurokapsiar
+
+## Summary
+
+Fluent UI components are designed to support various aspects of accessibility, so that they can be used with different input methods (mouse, touch, keyboard, screen readers) as well as fulfill different rendering and layout requirements (theming, zoom, contrast).
+
+## Background
+
+[Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/TR/WCAG21/) and [WAI-ARIA Authoring Practices](https://www.w3.org/TR/wai-aria-practices-1.2/) are community driven standard for accessible web pages and applications. They provide basic framework allowing consistent usage of web pages and applications with or without assistive technologies.
+
+## Problem statement
+
+Fluent UI components aim to fully respect both WCAG and ARIA and the Fluent UI team is working with the community to improve them. From the technical point of view, thee main challenges were identified:
+
+1. Accessibility is usually not intuitive as it is not immediately obvious. It is hard to achieve high quality user experience and easy to regress it.
```

Comment:
I completely get what this point is trying to capture, though I think the current wording risks implying that the only perspective we're speaking to is an abled one. That is to say, accessibility is very obvious and pretty intuitive for someone who is disabled. Maybe it could be reworded a bit to still capture the challenge without the implication?

One option could be something like this:
"Accessibility is a complex field that is often not taught and left behind in planning, design, development, and testing. It can be hard to achieve high quality user experiences and easy to regress them."